docs/examples/tooltip/virtual-trigger.vue

```diff
@@ -16,26 +16,17 @@ import { onMounted, onUnmounted, ref } from 'vue'
 
 const visible = ref(false)
 const triggerRef = ref({
-  getBoundingClientRect() {
-    return position.value
-  },
+  getBoundingClientRect: () => position.value,
 })
+const position = ref(DOMRect.fromRect())
```

Comment:
> rendering pages...ReferenceError: DOMRect is not defined

This causes SSR, SSG errors because `DOMRect` does not exist at server-side rendering.